src/core/annotation.js

```diff
@@ -721,13 +778,24 @@ class Annotation {
     }
   }
 
-  getBorderAndBackgroundAppearances() {
+  getBorderAndBackgroundAppearances(annotationStorage) {
```

Comment:
This is existing code, but the question above (regarding `WidgetAnnotation`) applies here as well.